### PR TITLE
Rendering stuff

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Dedicated to those who spent hours tirelessly reviewing #99 
@DeadlyMC 

~~i need this for fapi pls merge~~

### CODE REVIEW

File: mappings/net/minecraft/class_2152.mapping [hunk 1]
```diff
@@ -1,4 +0,0 @@
-CLASS net/minecraft/class_2152
```
Comment (DeadlyMC) [inline mappings/net/minecraft/class_2152.mapping:1]: Can you revert this? sky changes stuff in this class in his pr too.

Comment (DeadlyMC) [reply mappings/net/minecraft/class_2152.mapping:1]: You know what, will do that myself.

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/class_2152.mapping:1]: ask sky
i did it first :hahayes:

Comment (DeadlyMC) [reply mappings/net/minecraft/class_2152.mapping:1]: His pr is more huge, so doing it here is better

**Review by DeadlyMC — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Pull request merged**